ci/cd/Jenkinsfile_cd_pipeline

```diff
@@ -0,0 +1,69 @@
+// -*- mode: groovy -*-
+
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// Jenkins pipeline
+// See documents at https://jenkins.io/doc/book/pipeline/jenkinsfile/
+
+// MXNet Continuous Delivery Pipeline
+// Orchestrates the release of the different artifact by calling downstream release jobs.
+
+pipeline {
+  agent {
+    label 'restricted-utility'
```

Comment:
NODE_LABELS

Reply:
Actually, it's the same chicken/egg problem as with the CI pipelines, where we have. e.g.:

```
node('utility') {
  // Loading the utilities requires a node context unfortunately
  ...
}
```

Could we leave it like this. I don't think it would affect anything if I put it on non-restricted.

Reply:
Ok